### PR TITLE
Remove newline characters from output

### PR DESCRIPTION
**Pull request by andrewkatz**
Looks like they are breaking the next-semver action:

```
Fatal error: Uncaught Version\Exception\InvalidVersionString: Version string 'v1.0.0
' is not valid and cannot be parsed in /workdir/vendor/nikolaposa/version/src/Exception/InvalidVersionString.php:16
Stack trace:
#0 /workdir/vendor/nikolaposa/version/src/Version.php(65): Version\Exception\InvalidVersionString::notParsable('v1.0.0\r')
#1 /workdir/src/Next.php(25): Version\Version::fromString('v1.0.0\r')
#2 /workdir/next.php(14): WyriHaximus\Github\Actions\NextSemVers\Next::run('v1.0.0\r', true)
#3 /workdir/next.php(17): {closure}()
#4 {main}
  thrown in /workdir/vendor/nikolaposa/version/src/Exception/InvalidVersionString.php on line 16
```

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -29,8 +29,8 @@ exec(`git for-each-ref --sort=-creatordate --count 1 --format="%(refname:short)"
 
         console.log('\x1b[32m%s\x1b[0m', `Found tag: ${tag}`);
         console.log('\x1b[32m%s\x1b[0m', `Found timestamp: ${timestamp}`);
-        fs.appendFileSync(process.env.GITHUB_OUTPUT, `tag=${tag}\r\n`);
-        fs.appendFileSync(process.env.GITHUB_OUTPUT, `timestamp=${timestamp}\r\n`);
+        fs.appendFileSync(process.env.GITHUB_OUTPUT, `tag=${tag}`);
+        fs.appendFileSync(process.env.GITHUB_OUTPUT, `timestamp=${timestamp}`);
```
Comment (WyriHaximus) [inline main.js:33]: This should do the trick: 

```suggestion
        fs.appendFileSync(process.env.GITHUB_OUTPUT, `tag=${tag}\n`);
        fs.appendFileSync(process.env.GITHUB_OUTPUT, `timestamp=${timestamp}\n`);
```

Comment (andrewkatz) [reply main.js]: Updated.

**Review by WyriHaximus — APPROVED** [review]
Thanks for the quick fix :+1:

**Comment by andrewkatz** [comment]
Happy to help, thanks for responding quickly.

**Pull request merged**